pkg/asset/manifests/aws/cluster.go

```diff
@@ -31,6 +31,14 @@ func GenerateClusterAssets(ic *installconfig.InstallConfig, clusterID *installco
 		return nil, fmt.Errorf("failed to get user tags: %w", err)
 	}
 
+	sshRuleCidr := []string{"0.0.0.0/0"}
+	if ic.Config.Publish == types.InternalPublishingStrategy {
```

Comment:
```suggestion
	if ic.Config.Publish != types.ExternalPublishingStrategy {
```
So it's also restricted for Mixed strategy

Reply:
addressed